CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
-cmake_minimum_required(VERSION 3.10) # CMP0083 NEW
+cmake_minimum_required(VERSION 3.11) # CMP0083 NEW

```

Comment:
Could we limit this to only the DISCORD_INTEGRATION clause, I belive both android studio and visual studio 2017 are limited to 3.10

Reply:
From what I can tell:

* Visual Studio 2017 was already at CMake 3.12 in 2018
* Android Studio is at CMake 3.18.1 as of version 4.2